app.js

```diff
@@ -5,11 +5,11 @@ var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
```

Comment:
I suggest using es6 'const' instead of 'var'. If you expect your variable to be reassigned to smth else you can use 'let'. That goes for all of the rest code you wrote.